pressbooks/pressbooks-book
# Fix link to spinkit css

After merge https://github.com/pressbooks/pressbooks-book/pull/817 and tested locally, looks like I had a npm cache issue, because everything worked, but after remove npm modules and re-install everything, I realized it didn't.

The problem is Spinkit moved scss to .min.css inclusion, see: https://github.com/tobiasahlin/SpinKit/releases/tag/v2.0.0

> New: Added spinkit.min.css
> Changed: No more SCSS. All configuration is now done through CSS vars.

This change add the correct file to use Spinkit.

### Test
- Run: `npm install && npm run build`
- Clone a book and enable: Appearance > Theme Options > Web Options >  Enable comparison
- Edit a chapter 
- Open that chapter, and a new button should be displayed: "Show comparison with original"
- The comparison table should be displayed.

This library is used when large comparison need to be loaded an a spinner will appear until the comparison is loaded completely.
